io.github.jazztickets.irrlamb.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.jazztickets.irrlamb</id>
+	<name>irrlamb</name>
+	<summary>3D physics game in which you control a blue ball of energy in an attempt to touch all glowing orbs</summary>
+	<description>
+		<p>Find all the orbs in this challenging 3D physics game, using either mouse/keyboard or gamepad.</p>
+		<p>35 levels, including 15 tutorials, 7 "campaign" levels, 4 skateparks, 7 challenges, and 2 secrets.</p>
+		<p>Keep track of your fastest times and save replays.</p>
+	</description>
+	<categories>
+		<category>Game</category>
+		<category>ArcadeGame</category>
+	</categories>
+	<content_rating/>
+	<url type="homepage">https://github.com/jazztickets/irrlamb</url>
+	<url type="help">https://jazztickets.github.io/docs/irrlamb/</url>
+	<url type="bugtracker">https://github.com/jazztickets/irrlamb/issues</url>
+	<project_license>GPL-3.0</project_license>
```

Comment:
this SPDX identifier is invalid see https://spdx.dev/licenses/

Reply:
Worked with `org.freedesktop.appstream-glib validate`, but I'll change it to `GPL-3.0-only`